Makefile

```diff
@@ -0,0 +1,21 @@
+IMAGE_REGISTRY ?= quay.io/openstack-k8s-operators
+IMAGE_TAG ?= latest
+
+# MUST_GATHER_IMAGE needs to be passed explicitly to avoid accidentally pushing to openstack-k8s-operators/must-gather
+ifndef MUST_GATHER_IMAGE
```

Comment:
Right now, if we don't define `MUST_GATHER_IMAGE` we can't run `make check` . 

~~~
$ make check
Makefile:6: *** MUST_GATHER_IMAGE is not set..  Stop.
~~~

Maybe we can move this condition elsewhere so that when some run make check` or any other target that don't need ` MUST_GATHER_IMAGE` they don't hit this issue

Reply:
Done